<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestGestionDuStock</class>
 <widget class="QWidget" name="TestGestionDuStock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TestGestionDuStock</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>231</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Entrée un rouleau</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Référence : </string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QPushButton" name="pushButtonNouveau">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Ajouter un nouveau</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Diamètre : </string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEditReference"/>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEditDiametre"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonQuitter">
      <property name="text">
       <string>Quitter</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>70</y>
     <width>191</width>
     <height>321</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Rouleaux en stock</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="listWidgetStock"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonRetrait">
      <property name="text">
       <string>Demander retrait</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>70</y>
     <width>211</width>
     <height>321</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Rouleaux retenus</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="listWidgetRouleauxRetenus"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonSupprimer">
      <property name="text">
       <string>Supprimer du stock</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>430</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButtonEnregistrer">
      <property name="text">
       <string>Enregistrer</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonRecuperer">
      <property name="text">
       <string>Récupérer</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonQuitter</sender>
   <signal>clicked()</signal>
   <receiver>TestGestionDuStock</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>382</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>481</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
